crypto/awskms/awskms.go

```diff
@@ -0,0 +1,170 @@
+/*
+ * Flow Go SDK
+ *
+ * Copyright 2019 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Package awskms provides a AWS Key Management Service (KMS)
+// implementation of the crypto.Signer interface.
+//
+// The documentation for AWS KMS can be found here: https://docs.aws.amazon.com/kms/index.html
+package awskms
+
+import (
+	"context"
+	"encoding/pem"
+	"fmt"
+	"strings"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	kms "github.com/aws/aws-sdk-go-v2/service/kms"
+	"github.com/aws/aws-sdk-go-v2/service/kms/types"
+	"github.com/onflow/flow-go-sdk/crypto"
+)
+
+const (
+	resouceArnFormat        = "arn:aws:kms:%s:%s:key/%s"
+)
+
+// Client is a client for interacting with the AWS KMS API
+// using types native to the Flow Go SDK.
+type Client struct {
+	client *kms.Client
+}
+
+// Key is a reference to a AWS KMS asymmetric signing key.
+type Key struct {
```

Comment:
No need to export type and its' functions.

Reply:
Key needs to be exported to be used to create the key. I follow the existing pattern for the cloudkms. Will lowercase some other functions that are not required to be exported